### PR TITLE
Feat/redis demo

### PR DESCRIPTION
**Pull request by m1ghtym0**

### CODE REVIEW

File: samples/graphene-redis/README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+# Demo of graphene + marblerun
+
+First, ensure you have access to a Kubernetes cluster with SGX-enabled nodes and kubectl installed and configured.
+Probably the easiest way to get started is to run Kubernetes on an [Azure Kubernetes Service (AKS)](https://docs.microsoft.com/en-us/azure/confidential-computing/confidential-nodes-aks-get-started), which offers SGX-enabled nodes.
+
+* Install Marblerun on the Clutser
```
Comment (daniel-weisse) [inline samples/graphene-redis/README.md:6]:
```suggestion
* Install Marblerun on the Clutser
```
* Install Marblerun on the Cluster

File: samples/graphene-redis/.gitignore [hunk 2]
```diff
@@ -0,0 +1,355 @@
+## Ignore Visual Studio temporary files, build results, and
```
Comment (daniel-weisse) [inline samples/graphene-redis/.gitignore:1]: This file can be removed now

**Review by daniel-weisse — APPROVED** [review]

File: samples/graphene-redis/README.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+cp ./redis-server.manifest.template /graphene/Examples/redis/redis-server.manifest.template
+```
+
+Next we can build the Docker image:
+
+```bash
+docker build --tag ghcr.io/edgelesssys/redis-graphene-marble -f ./Dockerfile /graphene
+```
+
+The current Dockerfile is poorly optimized and can probably be configured to be smaller and more efficient, but for the sake of this demo it works
```
Comment (thomasten) [inline samples/graphene-redis/README.md:109]: Can we rephrase it to sound a bit nicer?

**Review by thomasten — CHANGES_REQUESTED** [review]

File: samples/graphene-redis/README.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
+# Demo of graphene + marblerun
+
+First, ensure you have access to a Kubernetes cluster with SGX-enabled nodes and kubectl installed and configured.
```
[the remark this answers is not in the document]
Comment (m1ghtym0) [reply samples/graphene-redis/README.md]: Not sure, can Graphene run in simulation mode, or does it require rebuilding?

[the remark this answers is not in the document]
Comment (thomasten) [reply samples/graphene-redis/README.md]: Don't know. Let's just keep it as is and don't waste time trying.

File: samples/graphene-redis/README.md [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+    localhost:6379> get mykey
+    "somevalue"
+    ```
+
+## Building the Docker image
+
+*Prerequisite*: Graphene is set up and the original redis sample is working.
+
+To marbleize the sample we edited [redis-server.manifest.template](redis-server.manifest.template). See comments starting with `MARBLERUN` for explanations of the required changes.
+Assuming you have built Graphene in `/graphene` copy the redis-server.manifest.template into the `/graphene/Examples/redis`
```
Comment (thomasten) [inline samples/graphene-redis/README.md:114]:
```suggestion
To marbleize the sample we edited [redis-server.manifest.template](redis-server.manifest.template). See comments starting with `MARBLERUN` for explanations of the required changes.

Assuming you have built Graphene in `/graphene` copy the redis-server.manifest.template into the `/graphene/Examples/redis`
```
suggest new paragraph

File: samples/graphene-redis/README.md [hunk 6]
```diff
@@ -0,0 +1,124 @@
...
+    OK
+    localhost:6379> get mykey
+    "somevalue"
+    ```
+
+## Building the Docker image
+
+*Prerequisite*: Graphene is set up and the original redis sample is working.
+
+To marbleize the sample we edited [redis-server.manifest.template](redis-server.manifest.template). See comments starting with `MARBLERUN` for explanations of the required changes.
```
Comment (thomasten) [inline samples/graphene-redis/README.md:113]:
```suggestion
To marbleize the sample we edited [redis-server.manifest.template](redis-server.manifest.template).
```
either remove the sentence or add these comments

**Review by thomasten — CHANGES_REQUESTED** [review]

**Review by thomasten — APPROVED** [review]
